pulp_container/tests/functional/api/test_push_content.py

```diff
@@ -37,30 +37,39 @@ def setUpClass(cls):
         cls.user_creator = gen_user(
             [
                 "container.add_containerdistribution",
-                "container.view_containerdistribution",
                 "container.add_containernamespace",
             ]
         )
-        cls.user_collaborator = gen_user(
+        cls.user_dist_collaborator = gen_user(
+            [
+                "container.pull_containerdistribution",
+                "container.push_containerdistribution",
+                "container.view_containerpushrepository",
+            ]
+        )
+        cls.user_namespace_collaborator = gen_user(
             [
                 "container.add_containerdistribution",
-                "container.view_containerdistribution",
-                "container.manage_namespace_distributions",
+                "container.pull_containerdistribution",
+                "container.push_containerdistribution",
                 "container.view_containerpushrepository",
+                "container.manage_namespace_distributions",
             ]
         )
         cls.user_reader = gen_user(["container.view_containerpushrepository"])
         cls.user_helpless = gen_user([])
 
-        cls._pull("centos:7")
```

Comment:
any reason we were pulling such a big image?

Reply:
I think that was because the test orinigally meant to be triggering multiple upload chunks.

Reply:
We may restore the "old" test and repurpose this one for RBAC and use a tiny image.